src/ext/java/org/opentripplanner/ext/emissions/EmissionsDataService.java

```diff
@@ -0,0 +1,42 @@
+package org.opentripplanner.ext.emissions;
+
+import jakarta.inject.Inject;
+import java.util.Optional;
+import org.opentripplanner.framework.lang.Sandbox;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+@Sandbox
+public class EmissionsDataService implements EmissionsService {
```

Comment:
Maybe call this class `DefaultEmissionsService` as that seems to be way our standard naming convention.